src/control/mobile_manipulation/robots_world/tiago/tiago.urdf.xacro

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0"?>
```

Comment:
I am not sure that this file can be included in the toolkit. Perhaps PAL robotics can be contacted if we need to change the license.

Reply:
I am happy to contact PAL. What would the desired resolution be in this case, i.e. is the question whether we would be allowed to change the licence on this file to Apache 2.0 or simply whether PAL gives us the permission to include this file with this license?

Reply:
I think we need this file with Apache 2.0 license in order to be included in the toolkit. So  most probably we need to either change the license or a version of the file with a different license.  I think this shouldn't be an issue, since PAL is a partner in this project.

Reply:
I have contacted PAL robotics about this